integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -1018,9 +1055,12 @@ private void initialize(String inputYaml) throws Exception {
     domainMap.values().removeIf(Objects::isNull);
 
     // create config map and secret for custom sit config
-    if (domainMap.get("configOverrides") != null) {
+    if ((domainMap.get("configOverrides") != null)
+        && (domainMap.get("configOverridesFile") != null)) {
       // write hostname in config file for public address
 
+      String configOverridesFile = domainMap.get("configOverridesFile").toString();
+
```

Comment:
Why does this test change need to exercise custom config overrides?
The primary focus is testing whether the NAP has been exposed via a service, and the service port 'works' (actually redirects to the NAP).

Reply:
@tbarnes-us It doesn't. That is a refactoring I wrote to make it dynamic. I was using a custom file earlier. But that has since been removed. Only the flexibility refactoring remains.

Reply:
@tbarnes-us I thought it was a nice addition, but I still have to merge this with @vanajamukkara 's refactoring changes.